docs(docs-infra): remove unnecessary CSS units from zero values

MeAkib opened the pull request:
Removes redundant units like `px`, `rem`, and `em` from zero values across stylesheets. This simplifies the code and aligns with CSS best practices since zero values don't require units.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/contributing-docs/commit-message-guidelines.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.dev application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

JeanMeche reviewed (changes requested):
Looks like this is breaking some tests. Can you PTAL

MeAkib committed:
docs(docs-infra): remove unnecessary CSS units from zero values

Removes redundant units like `px`, `rem`, and `em` from zero values across stylesheets. This simplifies the code and aligns with CSS best practices since zero values don't require units.
